VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -360,9 +362,11 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
     toggleHandRaised() {
         if (this.handRaised) {
             this.videoCallService.lowerHand(this.conferenceId, this.participant.id);
+            this.videoControlService.setHandRaiseStatusById(this.participant.id, !this.handRaised);
```

Comment:
This and 369 could go outside the if else.

Reply:
Done